src/components/business/pendingqueueitem.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+class PendingQueueItem extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+ 
+  updateStatusToCurrent(item) {
+    // call to API to update status of order from 'pending' to 'current'
+    console.log('The status of ', item.OrderId, ' has been updated to "current."');
+  }
+  
+  render() {
+    return (
+      <div className="pending-orders">
+        <h5>Pending Queue Items</h5>
+        <div className="order-item">
+          {this.props.order.map((item, index) =>
+            <div key={index}>
```

Comment:
@jpdaprato you may already know this, but passing the index in as a key is an anti-pattern despite lots of people doing it. Take a look at https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318